style.css

```diff
@@ -0,0 +1,95 @@
+* {
```

Comment:
all the css is great and love that you're experimenting with it, just one thing:

no more white background and pink box for blackjack please LOL

also can look into css flexbox and grid to learn more